frontend/src/metabase/home/components/EmbedHomepage/InteractiveTabContent.tsx

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line no-restricted-imports
+import { List } from "@mantine/core";
+import { Link } from "react-router";
+import { t, jt } from "ttag";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Anchor, Button, Icon, Text } from "metabase/ui";
+
+import type { EmbedHomepageViewProps } from "./EmbedHomepageView";
+
+export const InteractiveTabContent = (props: EmbedHomepageViewProps) => {
+  const {
+    embeddingAutoEnabled,
+    interactiveEmbeddingQuickstartUrl,
+    exampleDashboardId,
+    licenseActiveAtSetup,
+    learnMoreInteractiveEmbedUrl,
+  } = props;
```

Comment:
If we don't need to spread props like in `frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepageView.tsx` what do you think about destructuring these props in the function arguments instead for consistency?

Reply:
19d24864a269e9be5fcb51da58c772a21976d1ee ✅